frontend/components/snippet.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import ClickToSelect from '@mapbox/react-click-to-select'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
```

Comment:
These changes ended up not being used and could possibly be reset. They're related to showing the full pattern (see comment in `markup-modal-content.js`).